test/unit/file-handlers/html-test.js

```diff
@@ -322,6 +359,40 @@ describe('Html handler', function () {
 			}).catch(done);
 		});
 
+		it('should not replace the replace the sources of img tags with srcset attribute, for which loadResource returned null', function (done) {
```

Comment:
Typo: redundant **the replace**

Should be `should not replace the sources of img tags with srcset attribute, for which loadResource returned null`